lib/src/client/options.dart

```diff
@@ -59,6 +59,6 @@ class ChannelOptions {
     this.userAgent = defaultUserAgent,
     this.backoffStrategy = defaultBackoffStrategy,
     this.connectionTimeout = defaultConnectionTimeOut,
-    this.codecRegistry,
+    this.codecRegistry = const CodecRegistry.empty(),
```

Comment:
This seems like a change in behavior: previously we would not send any `grpc-accept-encoding`, now we would send identity instead (given the change that `empty` means `identity` now).

Reply:
Restored original behavior